src/cgame/cg_consolecmds.cpp

```diff
@@ -579,9 +579,10 @@ void CG_InitConsoleCommands()
 		trap_AddCommand( commands[ i ].cmd );
 	}
 
+	// Defined in src/engine/qcommon/q_shared.h, see BUTTON_ATTACK etc.
 	trap_RegisterButtonCommands(
-	    // 0      12       3     45      6        78       9ABCDEF      <- bit nos.
-	      "attack,,useitem,taunt,,sprint,activate,,attack2,,,,,,rally"
+	    // 0      1       2       3     45      6        7       89ABCDE     F <- bit nos.
+	      "attack,attack2,attack3,taunt,,sprint,activate,useitem,,,,,,,rally,"
```

Comment:
useitem should just be deleted, right?

Reply:
Yes, but at this point it is done iteratively, `attack3` needs to be added before we can drop `useitem`, and it's better to put `attack3` at the right place since the beginning, hence the temporary move of `useitem` to keep each commit usable before being able to drop it entirely.